src/adam_impact_study/conversions.py

```diff
@@ -162,75 +119,144 @@ def sorcha_output_to_od_observations(sorcha_observations_df):
                 ("origin.code", "ascending"),
             ]
         )
-        od_observations[obj] = OrbitDeterminationObservations.from_kwargs(
-            id=[f"{obj}_{i}" for i in range(len(object_obs))],
+
+        od_observation = Observations.from_kwargs(
+            obs_id=[f"{obj}_{i}" for i in range(len(object_obs))],
+            object_id=[obj for i in range(len(object_obs))],
```

Comment:
```suggestion
            object_id=pa.repeat(obj, len(object_obs)),
```
Not required, but I like that it puts it immediately into a pyarrow array and is more concise than the list comprehension.